recipes/ludoSim/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "ludoSim-jaib1" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: e04071d847996209c3e0485dd814207619c3c3e3d404894fa531d03622f8c8d5
+  - url: https://raw.githubusercontent.com/jaib1/ludoSim/master/LICENSE
+    sha256: 3972dc9744f6499f0f9b2dbf76696f2ae7ad8af9b23dde66d6af86c9dfb36986
+
+build: 
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: true  # [py<30]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - ludoSim
+  requires:
+    - pytest
+  source_files:
```

Comment:
Hi @isuruf 

I am getting the error: 
```
RuntimeError: Did not find any source_files for test with pattern tests/test_ludo.py
```

I have the `tests` folder in the root of my package repository, which is what I presume is the "source work directory" mentioned in the docs. Do you know why the builder isn't seeing my `tests` folder?

Reply:
Hi @isuruf, following up on this

Reply:
It seems like they are in the `ludoSim/` directory: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=56480&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&taskId=bef775d0-a1a2-55a3-9721-17147029baf0&lineStart=489&lineEnd=490&colStart=1&colEnd=1

ie `ludoSim/tests/test_ludo.py`

Reply:
I don't think that's the issue, as I just tried that and [got the same failure](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=60957). Also, that `tests` folder is redundant in both the root directory and the nested `ludoSim` directory because I tried seeing if this was an issue earlier.